badge_server/main.py

```diff
@@ -42,12 +42,12 @@
 
 import utils as badge_utils
 from compatibility_lib import utils as compat_utils
-from compatibility_lib.compatibility_store import Status as PackageStatus
-from compatibility_lib.dependency_highlighter import PriorityLevel
+from compatibility_lib import dependency_highlighter as highlighter
```

Comment:
Unfortunately, you can't do this either:
https://github.com/google/styleguide/blob/gh-pages/pyguide.md#224-decision

Reply:
I think at >20 characters, 'dependency_highlighter' would count as inconveniently long?